api/src/main/resources/workbench.yaml

```diff
@@ -354,6 +354,25 @@ paths:
                 schema:
                   $ref: "#/definitions/CriteriaListResponse"
 
+  /api/v1/cohort-builder/search:
```

Comment:
As mentioned to Jon and Brian previously: returning a full list of participant IDs for each set of criteria and intersecting them in the browser isn't going to scale well... it would be better to have BigQuery do all the work (which it is really good at), and return just counts to the browser for the cohort building screen; and when returning data on actual participants (e.g. in the review tab), use pagination to ensure we're not pulling down more data than we need.

I know y'all want to test out the performance characteristics of this approach before you head that direction -- let's just make sure we do that soon, before too much gets invested in this approach.